src/server.ts

```diff
@@ -15,29 +14,29 @@ const HOST = '0.0.0.0';
 const app = express();
 const parser = json();
 
-app.get("/ping", (req, res) => {
-  res.status(200).send();
+app.get("/", (req:any, res:any) => {
+  res.status(200).send("hello world!");
+});
+
+app.get("/ping", (req:any, res:any) => {
+  res.status(200).send("pong");
 });
 
-app.put('/local-item', parser, putItem);
 app.get('/local-item', parser, getItem);
```

Comment:
let's add a new route with the get function 

app.get('/local-item/:id', parser, getItem);
app.get('/local-item', parser, listItems);